de.metas.acct.base/src/main/sql/postgresql/ddl/functions/acctBalanceUntilDate.sql

```diff
@@ -1,12 +1,36 @@
 -- Function: de_metas_acct.acctbalanceuntildate(numeric, numeric, date, character)
 
- DROP FUNCTION IF EXISTS de_metas_acct.acctbalanceuntildate(numeric, numeric, date, character);
+DROP FUNCTION IF EXISTS de_metas_acct.acctbalanceuntildate(numeric, numeric, date, character);
 
-DROP FUNCTION IF EXISTS   de_metas_acct.acctbalanceuntildate(p_account_id numeric, p_c_acctschema_id numeric, p_dateacct date,  ad_org_id numeric, p_includepostingtypestatistical character);
-DROP FUNCTION IF EXISTS   de_metas_acct.acctbalanceuntildate(p_account_id numeric, p_c_acctschema_id numeric, p_dateacct date,  ad_org_id numeric, p_includepostingtypestatistical character, IN  p_ExcludePostingTypeYearEnd character);
+DROP FUNCTION IF EXISTS de_metas_acct.acctbalanceuntildate(p_account_id numeric, p_c_acctschema_id numeric, p_dateacct date, ad_org_id numeric, p_includepostingtypestatistical character);
+DROP FUNCTION IF EXISTS de_metas_acct.acctbalanceuntildate(p_account_id numeric, p_c_acctschema_id numeric, p_dateacct date, ad_org_id numeric, p_includepostingtypestatistical character, IN p_ExcludePostingTypeYearEnd character);
 
+--
+--
+--
+--
+--
+-- TODO this will be deleted in https://github.com/metasfresh/metasfresh/issues/6149
+--      DO NOT USE THIS ANYMORE
+--      PLEASE USE acctBalanceToDate
```

Comment:
it's great to provide this info, but this please also add it as a comment to the function in question, so future devs can see it there too

Reply:
Added comment to the function.